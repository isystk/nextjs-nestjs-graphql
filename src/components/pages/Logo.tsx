import React from 'react'
import Link from 'next/link'
import { URL } from '@/common/constants/url'

const Logo: React.FC = () => {
  return (
    <div className="header-logo">
      <Link href={URL.HOME}>
        <a style={{ fontSize: '25px', color: '#fff' }}>
          {process.env.APP_NAME}
        </a>
      </Link>
    </div>
  )
}

export default Logo
