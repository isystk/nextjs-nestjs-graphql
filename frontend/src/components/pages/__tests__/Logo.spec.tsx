import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import Logo from '../Logo'

test('Logo', () => {
  const wrapper = shallow(<Logo />)
  expect(toJson(wrapper)).toMatchSnapshot()
})
