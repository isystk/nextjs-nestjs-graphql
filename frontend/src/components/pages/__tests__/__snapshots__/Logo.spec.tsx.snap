// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Logo 1`] = `
<div
  className="header-logo"
>
  <Link
    href="http://localhost:3000/"
  >
    <a
      style={
        Object {
          "color": "#fff",
          "fontSize": "25px",
        }
      }
    />
  </Link>
</div>
`;
